app/assets/stylesheets/hitobito/customizable/_wagon.scss

```
@@ -0,0 +1,58 @@
+//  Copyright (c) 2012-2020, Stiftung für junge Auslandssschweizer. This file is part of
+//  hitobito_sjas and licensed under the Affero General Public License version 3
+//  or later. See the COPYING file at the top-level directory or at
+//  https://github.com/hitobito/hitobito_sjas.
+
+// Links (but not buttons) should be underlined
+a,
+.parent > .nav a:hover {
+  text-decoration: underline;
+}
+a.btn,
+.content-header a {
+  text-decoration: none;
+}
+
+// Uppercase text in buttons and navigation links
+.btn,
+.nav-left-list a,
+.nav-sub a {
+  text-transform: uppercase;
+  font-weight: bold;
+  letter-spacing: -0.03em;
+}
+
+// Default button color
+.btn {
+  color: $black;
+}
+.btn .caret {
+  border-top-color: $black;
+}
+
+// Active navigation element
+.nav-left {
+  a.is-active, li.is-active > a {
+    text-decoration: none;
+    background-color: $sjasRed;
+    color: $sjasWhite;
+    border-color: $sjasBlue4;
+
+    &:hover {
+      color: lighten($sjasRed, 46%);
+    }
+  }
+}
+
+// Minor fixing & brushing
+.nav-pills > .active > a {
+  border-color: $black;
+}
+
+.nav-left .nav-left-section:first-child {
+  border-color: $sjasGray1;
+}
```

```suggestion
}
#page-header > a {
  color: $black;
}
```

Und hier noch die farbliche Anpassung. Leider ist es im original so geschrieben, dass es auf einer Hintergrundfarbe basiert. Das richtig abstrakt zu lösen ist für mich gerade schwerer als hier nur den Override zu machen.

---

Danke, ganz korrekt die Links im Page Header sind mir durch die Lappen gegangen.